### ISSUE TITLE
[BUG]: Need to set Stratify's `param_renaming_uses_strata_names` to `True`

### ISSUE THREAD

**Issue by liunelson** [DARPA-ASKEM/terarium#4304]
Currently, Stratify Model creates new parameters with name like `<param_name>_0, <param_name>_1, ...` instead of using the strata names that the user provides.

In the most recent version of MIRA, Ben added a new argument to `mira.metamodel.ops.stratify` called `param_renaming_uses_strata_names` to toggle whether the newly created parameters are named `<param_name>_<strata_name_0>, <param_name>_<strata_name_1>, ...`

TLDR: we should call `mira.metamodel.ops.stratify(..., param_renaming_uses_strata_names = True)`

This might impact the UI parameter matrix generation?
